src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -25,6 +25,10 @@
  */
 abstract class FileLoader extends Loader
 {
+    const IGNORE_NONE = 0;
+    const IGNORE_ALL = 1;
+    const IGNORE_FILE_NOT_FOUND = 2;
```

Comment:
wouldn't it be more logical to have `IGNORE_ALL` have a greater value than `IGNORE_FILE_NOT_FOUND`

Reply:
Indeed. I have changed the values:
```
IGNORE_NONE = 0
IGNORE_FILE_NOT_FOUND = 1
IGNORE_ALL = 256 (big to reserve some lower numbers for future errors that we may ignore in the future)
```